modules/search/class.jetpack-search.php

```diff
@@ -186,13 +186,40 @@ public function init_hooks() {
 			add_action( 'init', array( $this, 'set_filters_from_widgets' ) );
 
 			add_action( 'pre_get_posts', array( $this, 'maybe_add_post_type_as_var' ) );
+			add_action( 'wp_enqueue_scripts', array( $this, 'load_assets' ) );
 		} else {
 			add_action( 'update_option', array( $this, 'track_widget_updates' ), 10, 3 );
 		}
 
 		add_action( 'jetpack_deactivate_module_search', array( $this, 'move_search_widgets_to_inactive' ) );
 	}
 
+	/**
+	 * Loads assets for Jetpack Search Prototype featuring Search As You Type experience.
+	 */
+	public function load_assets() {
```

Comment:
Will we want to load this on all pages by default, or only when a searchform is present?

Reply:
When activated then needs to be on all pages.